### PR TITLE
Add `net` subcommand

### PR DESCRIPTION
**Pull request by mkeeter**
The `net` subcommand allows for management and debugging of the _client side_ of the management network, i.e. the KSZ8463 + VSC85x2 which is common across all boards.

Depends on [hubris#709](https://github.com/oxidecomputer/hubris/pull/709)

# Todo
- [x] Bump version
- [x] Write some docs

### CODE REVIEW

File: cmd/net/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+
+    // Due to HIF limitations (lack of Expand16 / Collect16), we're going to
+    // limit ourselves to 255 MAC table entries.
+    let (mac_count, clipped) = if mac_count > u8::MAX as u32 {
+        (u8::MAX, true)
+    } else {
+        (mac_count as u8, false)
+    };
+
+    println!("Reading {} MAC addresses...", mac_count);
```
Comment (labbott) [inline cmd/net/src/lib.rs:135]: these `println` should be `humility::msg`

File: cmd/net/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+                println!("Skipping empty MAC address");
+            } else {
+                mac_table.entry(port).or_default().push(mac);
+            }
+        } else {
+            // Log the error but keep going for other entries in the table
+            println!("Got error result: {:?}", r);
+        }
+    }
+    println!(" {} |        {}", "PORT".bold(), "MAC".bold());
```
Comment (labbott) [inline cmd/net/src/lib.rs:193]: I wonder if it would be worth it to put our table printing into a library? (Not needed, more thinking long term)

Comment (mkeeter) [reply cmd/net/src/lib.rs:193]: Yeah, or maybe switching to something like [`tabled`](https://crates.io/crates/tabled)...

**Review by labbott — COMMENTED** [review]
LGTM!

**Pull request merged**